tasks/101_least-common-multiple/least-common-multiple.js

```diff
@@ -0,0 +1,11 @@
+export const leastCommonMultiple = (num1, num2) => {
+  let greaterNumber = Math.max(num1, num2);
+  let smallerNumber = Math.min(num1, num2);
+  let multiple;
+  for (let i = 1; i < 10; i++) {
```

Comment:
up to 10? maybe there is other algorithm to do it, probably there is a formula with using greatestCommonDivisor